README.md

```diff
@@ -1,35 +1,65 @@
-  --docpath                Path to document                                                     [string] [default: "/doc/doc/drugcases.qvf"]
-  --max, -m                Max number of sessions                                                                                   [number]
-  --interval, -i           How often new sessions should be created                                                   [number] [default: 60]
-  --selectionInterval, -s  How often selections should be done                                                     [number] [default: 10000]
-  --selectionRatio, -r     The amount of sessions the should do selections (in %)                                    [number] [default: 0.1]
-  --loginUrl, -l           If a cookie should be fetched and used in ws header
-                                                                [string] [default: "/login/local/callback?username=admin&password=password"]
-  --cookie                 Fixed cookie to be used in ws header                                                                     [string]
-  --keepAlive, -k          Don´t close sessions after ramp up                                                     [boolean] [default: false]
-  --objects                Defined objects to create after session create                                              [array] [default: []]
-  --secure                 Wheather to use wss or ws                                                               [boolean] [default: true]
-  --seed                   What seed to use for the random selections, if none is provided it will use a random GUID.
-                           The seed for each thread will be this string and the thread number              [string] [default: a random GUID]
-  --config, -c             Path to JSON config file                                                                 [string] [default: null]
-  ```
-
-Configurations can be specified in a config file and be used as only parameter to the `cli.js`
+  --help, -h             Show help                                                                                                 [boolean]
+  --threads, -t          Number of threads to run in parallell/n Setting to `-1` will use the number of cores          [number] [default: 1]
+  --max, -m              Max number of sessions                                                                                     [number]
+  --interactionInterval  How often interactions should be done (in ms)                                             [number] [default: 10000]
+  --interactionRatio     The amount of sessions the should do selections (in %)                                      [number] [default: 0.1]
+  --keepAlive, -k        Don´t close sessions after ramp up                                                       [boolean] [default: false]
```

Comment:
I find it hard to understand how `--keepAlive` works with `--sessionLength` and `--triangular`. Are all needed or could, for example, "keep-alive" be indicated by setting `--sessionLength` to <= 0?